src/Reflection/Adapter/ReflectionEnum.php

```diff
@@ -530,14 +530,18 @@ public function getCase(string $name): ReflectionEnumUnitCase|ReflectionEnumBack
     /** @return list<ReflectionEnumUnitCase|ReflectionEnumBackedCase> */
```

Comment:
Consideration here: we perhaps return `list<ReflectionEnumUnitCase>|list<ReflectionEnumBackedCase>` rather than `list<ReflectionEnumUnitCase|ReflectionEnumBackedCase>` :thinking:

Reply:
AFAIK phpstan is not able to differentiate it, but I can do it if you prefer

Reply:
It's more precise for the consumer regardless, no?